resources/database/collection_exercise_setup/rsi_201812_setup.sql

```diff
@@ -15,9 +15,14 @@ INSERT INTO collectionexercise.casetypedefault ( casetypedefaultpk, sampleunitty
 VALUES ( 1000003, 'BI', '90dd5b4f-381f-4766-a19b-a8346a7cdba1', '75b19ea0-69a4-4c58-8d7f-4458c8f43f5c' )
 ON CONFLICT (casetypedefaultpk) DO NOTHING;
 
-INSERT INTO action.actionrule ( actionrulepk, actionplanfk, actiontypefk, name, description, daysoffset, priority, id)
-VALUES ( 1000003, 1000002, 1, 'MBSENR', 'Enrolment', 0, 3, 'd24b3f17-bbf8-4c71-b2f0-a4334125d78b');
-INSERT INTO action.actionrule ( actionrulepk, actionplanfk, actiontypefk, name, description, daysoffset, priority, id)
-VALUES ( 1000004, 1000002, 2, 'MbSREM1', 'Enrolment Reminder', 45, 3, 'd24b3f17-bbf8-4c71-b2f0-a4334125d78c');
-INSERT INTO action.actionrule ( actionrulepk, actionplanfk, actiontypefk, name, description, daysoffset, priority, id)
-VALUES ( 1000005, 1000003, 3, 'MbSREM2', 'Survey Reminder', 45, 3, 'd24b3f17-bbf8-4c71-b2f0-a4334125d78d');
+INSERT INTO action.actionrule (id, actionrulepk, actionplanfk, actiontypefk, name, description, daysoffset, priority)
+VALUES ('205f3512-eb6e-419d-81fb-d563f8b27bb3', 1000003, 1000002, 1, 'MBSENR', 'Enrolment', 0, 3)
```

Comment:
Sorry they weren't duplicate ids, I was being lazy and just changed the last letter of each uuid. 😄